libraries/fault_sdk/fsInjection.c

```diff
@@ -7,20 +7,14 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <fm_util.h>
+#include <inttypes.h>
 #include "fsInjection.h"
 
-float fsInjection(int cmdToFaultSensor)
-{
-	/*Definition of Variables*/
-	float theta_hat_rel,theta_rel,delta_theta_rel;	
-	
-	if (cmdToFaultSensor == 1)
-	{
-		delta_theta_rel = 10; /*TODO change to better bias*/
-	}
-	else
-	{
-		delta_theta_rel = 0; /*No bias if not commanded to fault*/;
+
+void fsInjection(double *deltaThetaRad, FmState *fmState)
+{   
+	if (fmState->cmdToFaultFS){
+		*deltaThetaRad +- fmState->fsBias;
```

Comment:
I would think we'd want `+=` here. It's mostly a philosophical argument, but my thought is this: The assumption from someone less familiar with the system would be that setting a positive value for `fmState->fsBias` would lead to a positive bias, and a negative would lead to a negative bias. Maybe I'm forgetting something tho